### DISCUSSION TITLE
Add CMakeTools.json file to '.vscode' folder

### DISCUSSION THREAD

**Discussion by vkryman** [microsoft/vscode-cmake-tools#2832] [Ideas]
I'm working on different linux remote servers simultaneously where are a different environment and different versions of the CMake software.
As I know at the moment CMakeTools package has global settings only for all VSCode projects. 
It will be better if all CMakeTools settings will be located locally on remote servers and will be independent on each other...

**Comment by bobbrow** [comment]
I'm not sure exactly what you would want to `CMakeTools.json` to be.  Have you looked into using [CMake Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html)?  We have some additional documentation here as well: https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/cmake-presets.md

**Reply by vkryman** [reply]
At the moment I'm talking about cmake.cmakePath and other parameters from settings.json file of vscode.

**Reply by bobbrow** [reply]
There is already a `settings.json` file in the `.vscode` folder that can override any global settings such as `cmake.cmakePath` and apply them to the current workspace/folder.  Introducing CMakeTools.json wouldn't add any features beyong that.  Am I misunderstanding your request?

**Reply by vkryman** [reply]
Yes, you are right, settings.json file in the .vscode folder can override the global settings and it works in my case! Why I did not find the solution for CMakeTools before? Many thanks...
